docs/DEPENDENCIES.md

```diff
@@ -0,0 +1,122 @@
+# Dependencies for deploying harbor-operator-app
+## Pre-deployment dependencies
+### Operational requirements
+In order to deploy the harbor-operator-app in a management cluster (MC), you need to have the following available:
+1. S3 Buckets: 
```

Comment:
Do these buckets have to be pre-created or will the operators create them with the below permissions?

Reply:
They have to be pre-created, all the items under the Operational Requirement section have to be pre-created